core/requestlogger/logger.go

```diff
@@ -74,6 +91,10 @@ func (r *logger) Filter(ctx context.Context, req *web.Request, w http.ResponseWr
 	return &loggedResponse{
 		result: webResponse,
 		logCallback: func(rwl *responseWriterLogger) {
+			if r.trackResponseCount {
+				go recordResponseStatus(ctx, rwl.statusCode)
```

Comment:
Change of style here: why aren't you checking the config within `trackResponseCount` ?

Reply:
Done.